# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hermes/automake-fix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur Hermes-1.3.3-orig/configure.in Hermes-1.3.3/configure.in
--- Hermes-1.3.3-orig/configure.in	2008-03-09 19:00:05.000000000 +0000
+++ Hermes-1.3.3/configure.in	2008-03-09 19:02:11.000000000 +0000
@@ -148,6 +148,8 @@
 AR_COMPILER=$CC
 AR_FLAGS=$CFLAGS
 
+AM_PROG_AS
+
 dnl let's keep autoconf 2.5 happy
 AS=$CC
 AC_SUBST(AS)
